### ISSUE TITLE
Angular Example Tests are not running

### ISSUE THREAD

**Issue by fokusferit** [tastejs/todomvc#1616]
I've cloned the todomvc project and wanted to use the angularjs example for some "refresh" of my angular 1.x knowledge + testing.

The README says:

```
npm install
npm test
```

Which results in an overall fail of all 17 tests with this error:

```
Chrome 49.0.2623 (Mac OS X 10.11.3) todoFocus directive should focus on truthy expression FAILED
    Error: [$injector:modulerr] Failed to instantiate module todomvc due to:
    Error: [$injector:modulerr] Failed to instantiate module ngResource due to:
    Error: [$injector:nomod] Module 'ngResource' is not available! You either misspelled the module name or forgot to load it. If registering a module ensure that you specify the dependencies as the second argument.
    http://errors.angularjs.org/1.4.3/$injector/nomod?p0=ngResource
        at /Users/ftopcu/Desktop/Projekte/exercises/todomvc/examples/angularjs/node_modules/angular/angular.js:68:12
....
```

After trying some things out, like making all versions fix to 1.4.3 and not the current 1.5.3 (which doesn't changed the error), 

I've checked the karma.conf.js file and realized that the angular-resource.js is not included but as the todo-module requires ngResource I would expect that the karma.conf.js should have the following file included:

```
files: [
    'node_modules/angular/angular.js',
    'node_modules/angular-route/angular-route.js',
    'node_modules/angular-resource/angular-resource.js',
    'node_modules/angular-mocks/angular-mocks.js',
    'js/**/*.js',
    'test/unit/**/*.js'
    ],
```

Maybe I did something wrong or my Karma, Jasmine version is too new, but adding angular-resource.js fixed the above error.

So before I do a PR for this, I wanted to ask this should be added or maybe I'm just wrong :smile:

**Comment by a13ks3y** [comment]
Have the same issue, https://github.com/tastejs/todomvc/pull/1625 works for me, but don't passing CI for some reason

**Comment by kveretennicov** [comment]
I also ran into this issue today. My attempt to fix: #1745 (CI passed).
